Packages/io.chainsafe.web3-unity/Runtime/Scripts/UI/Resizer.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace ChainSafe.Gaming
+{
+    [RequireComponent(typeof(RectTransform))]
+    public class Resizer : MonoBehaviour
```

Comment:
What is the exact purpose of this? To make the modal fit all resolutions?

Reply:
yes, as a percentage of screen height, WebGL modal becomes scrollable when height is too low but stretched with taller screens. Resizes based on screen height percentage.